trunk/SUAVE/Methods/Weights/Buildups/eVTOL/empty.py

```diff
@@ -192,21 +197,6 @@ def empty(config,
         propulsor.avionics.origin[0][0]                                      = 0.4 * nose_length
         propulsor.avionics.mass_properties.center_of_gravity[0][0]           = 0.0
         propulsor.avionics.mass_properties.mass                              = output.avionics
-        
-        #-------------------------------------------------------------------------------
```

Comment:
Wait wait, did you delete the electronic speed controller weight for the environmental control system?

Reply:
I made a mistake - First I assigned ECS to ESC. then when I corrected it Jordan told me that it was in fact supposed to be ECS - environmental control system. There was never a weight build-up for ESC after all. This is of course why we always stress comments to escape this confusion.